packages/flutter/lib/src/cupertino/segmented_control.dart

```diff
@@ -179,6 +180,11 @@ class CupertinoSegmentedControl<T> extends StatefulWidget {
   /// Defaults to the selectedColor at 20% opacity if null.
   final Color pressedColor;
 
+  /// The [children], if any, are placed inside this padding.
+  ///
+  /// Defaults to EdgeInsets.symmetric(horizontal: 16.0)
+  final EdgeInsets padding;
```

Comment:
Let's change this to EdgeInsetsGeometry. Otherwise this is specifying that we use left and right regardless of Directionality.